src/drivers/mct/cime_config/config_component_cesm.xml

```diff
@@ -384,68 +384,16 @@
-      <value compset="^HIST_CAM4%FCHM"           >0.000001</value>
-      <value compset="^2000"                     >367.0</value>
-      <value compset="^1850.+DATM%NYF"           >379.000</value>
-      <value compset="^1850.+DATM%NYF.*_POP2%ECO">284.7</value>
-      <value compset="^2000.+DATM%NYF"           >379.000</value>
-      <value compset="^2000.+DATM%IAF"           >379.000</value>
-      <value compset="^HIST.+DATM"               >367.0</value>
-      <value compset="^4804.+DATM"               >367.0</value>
-      <value compset="^RCP2.+DATM"               >367.0</value>
-      <value compset="^RCP4.+DATM"               >367.0</value>
-      <value compset="^RCP6.+DATM"               >367.0</value>
-      <value compset="^RCP8.+DATM"               >367.0</value>
-      <value compset="^2000.+XATM"               >379.000</value>
-      <value compset="^HIST.+CAM"                >0.000001</value>
-      <value compset="^5505.+CAM"                >0.000001</value>
-      <value compset="^RCP2.+CAM"                >0.000001</value>
-      <value compset="^RCP4.+CAM"                >0.000001</value>
-      <value compset="^RCP6.+CAM"                >0.000001</value>
-      <value compset="^RCP8.+CAM"                >0.000001</value>
-      <value compset="^2013.+CAM"                >0.000001</value>
-      <value compset="^SDYN.+CAM"                >0.000001</value>
-      <value compset="^AMIP.+CAM"                >0.000001</value>
-      <value compset="^AR95"                     >368.9</value>
-      <value compset="^AR97"                     >368.9</value>
-      <value compset="^2003"                     >367.0</value>
-      <!-- Override values based on CAM (WACCM) chemistry -->
-      <value compset="CAM[456]0%W"               >0.000001</value>
-      <value compset="CAM[456]0%SVBS"            >0.000001</value>
-      <value compset="CAM[456]0%TMOZ"            >0.000001</value>
-      <!-- Mariana's fix to allow isotopes to get the proper CO2 value -->
-      <value compset="CAM..%WISOall"             >284.7</value>
-      <value compset="^2000.+CAM[45]%WCSC"       >368.9</value>
-      <value compset="^2000.+CAM[45]%MOZM"       >0.000001</value>
-      <value compset="^2000.+CAM[45]%MMOS"       >0.000001</value>
-      <value compset="^2000.+CAM[45]%TMOZ"       >0.000001</value>
-      <value compset="^2000.+CAM[45]%SMA3"       >0.000001</value>
-      <value compset="^2000.+CAM[45]%SMA7"       >0.000001</value>
-      <value compset="^2000.+CAM[45]%SSOA"       >0.000001</value>
-    </values>
-    <group>run_co2</group>
-    <file>env_run.xml</file>
-    <desc>This set the namelist values of CO2 ppmv for CAM and CLM. This variables is
-      introduced to coordinate this value among multiple components.</desc>
-  </entry>
-
-  <entry id="CO2_TYPE_COSTANT_VALUE">
-    <type>real</type>
-    <valid_values></valid_values>
     <default_value>284.7</default_value>
     <values>
       <value compset="^2000">367.0</value>
```

Comment:
Can we also have the following here?
      <value compset="DATM.*POP2%ECO">284.7</value>
I think this will produce 1850 values for any compset using datm and ocean BGC,
whether or not the compset name starts with 2000.